hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/Create2OperationSuite.java

```diff
@@ -515,10 +509,8 @@ private HapiSpec canMergeCreate2ChildWithHollowAccount() {
         final AtomicReference<AccountID> partyId = new AtomicReference<>();
         final AtomicReference<ByteString> partyAlias = new AtomicReference<>();
 
-        return propertyPreservingHapiSpec("CanMergeCreate2ChildWithHollowAccount")
-                .preserving(LAZY_CREATION_ENABLED)
+        return defaultHapiSpec("CanMergeCreate2ChildWithHollowAccount")
```

Comment:
Did Sonar suggest this change? It looks like more than a simple code-style change.

Reply:
This is related to [this comment](https://github.com/hashgraph/hedera-services/pull/7327#discussion_r1255853017) that is about the `CanMergeCreate2MultipleCreatesWithHollowAccount` spec below this one.

The change I did below was knowing that `CONTRACTS_NONCES_EXTERNALIZATION_ENABLED` will be enabled in https://github.com/hashgraph/hedera-services/pull/7425 so there is no need to enable it here anymore. While doing that I noticed that for `LAZY_CREATION_ENABLED` we have the same case and decided to remove it as well and by doing so those two specs will be less leaky i.e. they will not be updating the same properties file, and could be run in parallel in the future.